docs/rooosss/index.html

```diff
@@ -0,0 +1,102 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style/style.css">
+        <link rel="stylesheet" href="style/header.css">
+        <link rel="stylesheet" href="style/footer.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header">
```

Comment:
Are these white gaps ok for you?
I suggest moving the background-color to the `.header` and `.footer` correspondently.
![Screenshot at Dec 20 22-58-46](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/473d1d2c-244f-4d96-8d1d-fe81087d1ba0)